scripts/verifyChange.ts

```diff
@@ -0,0 +1,120 @@
+import util from 'node:util';
+
+type Options = {
+  fileLineChange: string;
+  diffPatchSeparated: string;
```

Comment:
On the one hand, I'm not a fan of having two separate inputs here with duplicated data. It means that technically, the script can be run with inputs that don't line up (i.e. you can pass in a file line change JSON that doesn't align with the patch that was provided and the script should handle these inconsistencies).

You can probably write a "poor man's diff parser", or just use something like https://www.npmjs.com/package/parse-diff to give a structured parsing of the diff.

I think the main advantage of going to JS/TS is that we have a lot more tools available and so we don't need to do similar hacks/short-cuts as we did in bash.

Reply:
Using `parse-diff` now 👍🏾